src/status_im/contexts/onboarding/syncing/progress/view.cljs

```diff
@@ -43,30 +42,29 @@
 (defn try-again-button
   [profile-color logged-in?]
   [quo/bottom-actions
-   {:actions (if logged-in? :one-action :two-actions)
+   {:actions (if logged-in? :one-action :two-vertical-actions)
     :blur? true
-    :button-one-label (i18n/label :t/recovery-phrase)
-    :button-one-props {:type                :primary
+    :container-style {:height (when-not logged-in? 116)}
```

Comment:
Can we give a name to this 116? 
I'm wondering what does it mean, I guess it's a sum of different UI elements because it's a big number. I also wonder if it's considering how it looks on different platforms.

Reply:
Thank you @ulisesmac for reviewing the PR. I have added a label here - https://github.com/status-im/status-mobile/pull/21890/files#diff-d6d12a107e3ffe116d992ea055af107e6b7c68f5f3084958999d739b01f15dfe